core/frontend/src/components/vehiclesetup/configuration/compass/ArdupilotMavlinkCompassSetup.vue

```diff
@@ -309,15 +325,64 @@ export default Vue.extend({
           (name) => autopilot_data.parameter(name),
         )
         if (offset_params.includes(undefined) || diagonal_params.includes(undefined)) {
-          results[compass.param] = false
+          results[compass.param] = {
+            calibration: CalibrationType.UNKNOWN,
+            color: 'red',
+            alert: 'error',
+            description: 'Unable to get calibration status, something went wrong.',
```

Comment:
```suggestion
            description: 'Unable to get calibration status - something went wrong.',
```
Should be either a dash, semi-colon, or full-stop+new sentence - comma doesn't really work there.